scripts/linux/trcE2ETest.sh

```diff
@@ -739,7 +743,7 @@ function run_longhaul_test() {
 
 	local hash
 	hash=$(tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c 8)
-	local device_id="$RELEASE_LABEL-Linux-$image_architecture_label-longhaul-$hash"
+	local device_id="$RELEASE_LABEL-Linux-$image_architecture_label-longhaul-$TRANSPORT_TYPE-$hash"
```

Comment:
It's not necessary since `hash` is already random.

Reply:
good point!